python/ambassador/envoy/v3/v3bootstrap.py

```diff
@@ -125,28 +127,93 @@ def __init__(self, config: 'V3Config') -> None:
                 name = 'envoy.stats_sinks.statsd'
                 typename = 'type.googleapis.com/envoy.config.metrics.v3.StatsdSink'
 
-            self['stats_sinks'] = [
-                {
-                    'name': name,
-                    'typed_config': {
-                        '@type': typename,
-                        'address': {
-                            'socket_address': {
-                                'protocol': 'UDP',
-                                'address': config.ir.statsd['ip'],
-                                'port_value': 8125
-                            }
+            stats_sinks.append({
+                'name': name,
+                'typed_config': {
+                    '@type': typename,
+                    'address': {
+                        'socket_address': {
+                            'protocol': 'UDP',
+                            'address': config.ir.statsd['ip'],
+                            'port_value': 8125
                         }
                     }
                 }
-            ]
+            })
 
             self['stats_flush_interval'] = {
                 'seconds': config.ir.statsd['interval']
             }
 
+        grpcSink = os.environ.get("AMBASSADOR_GRPC_METRICS_SINK")
```

Comment:
this environment variable is admittedly a hack (re: the convo that @rhs and i had about the release/v1.13 version of this change), and i'd really like to avoid doing "hacks" on 2.0. I saw in the convo yesterday some mention of CRDs, and yeah this should def get added to the ambassador module or something